ui/StatusQ/include/StatusQ/audioutils.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include <QObject>
+#include <QMediaPlayer>
+
+class QJSEngine;
+class QQmlEngine;
+
+class AudioUtils : public QObject
```

Comment:
I think we can drop this C++ class completely; the volume conversion is directly available from QML in both Qt5 and Qt6